common/src/types/v0/message_bus/nexus.rs

```diff
@@ -124,8 +124,8 @@ impl Default for NexusShareProtocol {
 impl From<i32> for NexusShareProtocol {
     fn from(src: i32) -> Self {
         match src {
-            0 | 1 => Self::Nvmf,
-            2 => Self::Iscsi,
+            0 => Self::Nvmf,
+            1 => Self::Iscsi,
```

Comment:
This no longer matches the protobuf file.

Reply:
Without this the volume protobuf in the control-plane is having issues. Need to figure out a way,

Reply:
What are the issues?

Reply:
https://github.com/openebs/mayastor-control-plane/blob/e061a7ce68705113541bbf4cecb1b6b9352db0e9/common/src/types/v0/message_bus/nexus.rs#L107

Here we map in this way, we don't have a none equivalent.

Reply:
https://github.com/openebs/mayastor-control-plane/blob/e061a7ce68705113541bbf4cecb1b6b9352db0e9/common/src/types/v0/message_bus/volume.rs#L119

Reply:
Hi, seems like we weren't using that. I removed it. Its not going to affect the same.